dataplane/saiserver/ports.go

```diff
@@ -426,7 +454,8 @@ func (l *lag) CreateLagMember(ctx context.Context, req *saipb.CreateLagMemberReq
 		Update: &fwdpb.PortUpdateDesc{
 			Port: &fwdpb.PortUpdateDesc_AggregateAdd{
 				AggregateAdd: &fwdpb.AggregatePortAddMemberUpdateDesc{
-					PortId: &fwdpb.PortId{ObjectId: &fwdpb.ObjectId{Id: fmt.Sprint(req.GetPortId())}},
+					InstanceCount: 1,
```

Comment:
What is InstanceCount?

Reply:
it's the number of copies of the port to add to the group (before this was adding 0 copies)

Reply:
Interesting, what does adding multiple copies of the same port to the group mean?

Reply:
you could unevenly weight the ports in the group i guess? 
send 75% of traffic to port 1 and 25% to port 2. Not sure that's practical use case for this

Reply:
Alright, if it passes the test and satisfy the use case you're going for then LGTM.